src/views/conference/2018/index/l10n.json

```diff
@@ -0,0 +1,25 @@
+{
+    "conference-2018.title": "Scratch Conference 2018: The Next Generation",
```

Comment:
I dithered a bit about the best way to do this one. Basically, we think it would look better on two lines with 'The Next Generation' starting on a new line.  You could add a `<br/>` in the string and use FormattedHTMLMessage, or split it into two strings (e.g., title and tagline), and just add the break in between the strings in the jsx.